src/scene/batching/batch.js

```diff
@@ -92,6 +93,16 @@ class Batch {
         this.meshInstance.aabb = this._aabb;
         this.meshInstance._aabbVer = 0;
     }
+
+    /**
+     * @deprecated
+     * @private
```

Comment:
This getter isn't private. It public, but presumably ignored from the documentation? So I suspect this should be `@ignore` instead.

Reply:
updated, good points